pkg/apis/serving/v1/revision_types.go

```diff
@@ -129,13 +129,22 @@ type RevisionStatus struct {
 	// +optional
 	LogURL string `json:"logUrl,omitempty"`
 
-	// ImageDigest holds the resolved digest for the image specified
+	// DeprecatedImageDigest holds the resolved digest for the image specified
 	// within .Spec.Container.Image. The digest is resolved during the creation
 	// of Revision. This field holds the digest value regardless of whether
 	// a tag or digest was originally specified in the Container object. It
 	// may be empty if the image comes from a registry listed to skip resolution.
+	// DEPRECATED Use ImageDigests instead.
+	// TODO(savitaashture) Remove deprecatedImageDigest in 2 releases(0.16).
```

Comment:
The deprecation timeline for stuff in v1 is a lot higher than this.

Reply:
Higher means which version?

Reply:
I'd just leave the note away on when it'll be removed 🤷‍♂ We can always decide that later.

Reply:
I'd follow the API deprecation guidance, which would leave these in for a year: https://kubernetes.io/docs/reference/using-api/deprecation-policy

Reply:
Modified 
Please take a look
Thank you